python/py-llvmlite/Portfile

```diff
@@ -28,44 +28,54 @@ if {${name} ne ${subport}} {
     legacysupport.newest_darwin_requires_legacy \
                         17
 
-    patchfiles-append   patch-ffi_Makefile.osx.diff
-
-    set llvmver "15"
-
     if {${python.version} == 39} {
         version             0.43.0
         checksums           rmd160  ea30be9de2f72c27635c4858968b95d1ec3ebaec \
                             sha256  ae2b5b5c3ef67354824fb75517c8db5fbe93bc02cd9671f3c62271626bc041d5 \
                             size    157069
-    }
 
-    post-patch {
-        reinplace "s|%%CXX%%|clang++-mp-${llvmver}|" ${worksrcpath}/ffi/Makefile.osx
+        set llvmver         "15"
+
+        depends_lib-append  port:llvm-${llvmver} \
+                            port:clang-${llvmver} \
+                            port:ncurses \
+                            port:libxml2
+        patchfiles-append   patch-ffi_Makefile.osx.diff
+
+        post-patch {
+            reinplace "s|%%CXX%%|clang++-mp-${llvmver}|" ${worksrcpath}/ffi/Makefile.osx
+
+            if {${os.major} <= 10} {
+                # https://trac.macports.org/ticket/61302
+                configure.ldflags-append \
+                            -framework CoreFoundation
+            } elseif {${os.major} >= 22} {
+                configure.ldflags-append \
+                            -lLLVM
+            }
 
-        if {${os.major} <= 10} {
-            # https://trac.macports.org/ticket/61302
-            configure.ldflags-append \
-                        -framework CoreFoundation
-        } elseif {${os.major} >= 22} {
-            configure.ldflags-append \
-                        -lLLVM
+            reinplace "s|%%MP_EXTRA_LDFLAGS%%|${configure.ldflags}|" ${worksrcpath}/ffi/Makefile.osx
         }
 
-        reinplace "s|%%MP_EXTRA_LDFLAGS%%|${configure.ldflags}|" ${worksrcpath}/ffi/Makefile.osx
-    }
+        build.env-append    LLVM_CONFIG=llvm-config-mp-${llvmver}                        
+        destroot.env-append LLVM_CONFIG=llvm-config-mp-${llvmver}
 
-    post-destroot {
-        system "/usr/bin/install_name_tool -change @rpath/libLLVM.dylib ${prefix}/libexec/llvm-${llvmver}/lib/libLLVM.dylib ${destroot}${python.pkgd}/llvmlite/binding/libllvmlite.dylib"
-    }
+        post-destroot {
+            system "/usr/bin/install_name_tool -change @rpath/libLLVM.dylib ${prefix}/libexec/llvm-${llvmver}/lib/libLLVM.dylib ${destroot}${python.pkgd}/llvmlite/binding/libllvmlite.dylib"
+        }
+    } else {
+        set llvmver         "20"
 
-    depends_lib-append  port:llvm-${llvmver} \
-                        port:clang-${llvmver} \
-                        port:zlib \
-                        port:ncurses \
-                        port:libxml2
+        depends_build-append \
+                            port:cmake
+
+        depends_lib-append  port:llvm-${llvmver} \
+                            port:zstd
+
+        build.env-append    LLVM_DIR=/opt/local/libexec/llvm-${llvmver}/lib/cmake/
```

Comment:
this should not hardcode `/opt/local` instead please use `${prefix}`

Reply:
Oops, fixed